### PR TITLE
[DependencyInjection] Add information about debugging tagged services

### PR DESCRIPTION
**Pull request by BOB41K1987**
Add documentations about debugging tagged services using `debug:container --tag`

### CODE REVIEW

File: service_container/debug.rst [hunk 1]
```diff
@@ -21,6 +21,21 @@ To see a list of all of the available types that can be used for autowiring, run
...
 
+How to Debug Service Tags & List Tagged Services
+------------------------------------
+You can find out what services are tagged with a specific tag using the following command:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:container --tag=kernel.event_listener
+
+Partial search is also available:
```
Comment (94noni) [inline service_container/debug.rst:32]: Perhaps document that it will prompt user with available matching tags

File: service_container/debug.rst [hunk 2]
```diff
@@ -21,6 +21,21 @@ To see a list of all of the available types that can be used for autowiring, run
...
+
+.. code-block:: terminal
+
+    $ php bin/console debug:container --tag=kernel
+
```
Comment (94noni) [inline service_container/debug.rst]: As it is a new feature, you must add a version added directive for 6.2

File: service_container/debug.rst (hunk 1 above)
Comment (BOB41K1987) [reply service_container/debug.rst:32]: Done.

File: service_container/debug.rst (hunk 2 above)
Comment (BOB41K1987) [reply service_container/debug.rst]: Done. I hope I did it correctly.

**Comment by javiereguiluz** [comment]
Thanks Vladimir ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**